python/tank/bootstrap/cached_configuration.py

```diff
@@ -178,6 +181,10 @@ def update_configuration(self):
             # Move to backup needs to undo changes when failing because we need to put the configuration
             # in a usable state.
             (config_backup_path, core_backup_path) = self._config_writer.move_to_backup(undo_on_error=True)
+            # Need the os.path.dirname call because this path of the backup config
+            # folder actually points to a 'config' subfolder of the backup folder
+            self.config_backup_folder_path = os.path.dirname(config_backup_path) if config_backup_path else config_backup_path
+            self.core_backup_folder_path = core_backup_path
```

Comment:
I am a bit unsure as to what you are getting at here. Could you elaborate? Thanks